static/loading-bar.js

```diff
@@ -10,18 +12,27 @@ function LoadingBar(color, parent) {
   return this;
 }
 
-LoadingBar.prototype.start = function(percentage) {
-  percentage = percentage || 0;
-  this.meter.css('width', percentage + '%');
+LoadingBar.prototype.start = function(progress) {
+  if(this.timer){
+    clearTimeout(this.timer);
+    this.timer = null;
+  }
+  this.meter.css('width', progress || starting + '%');
```

Comment:
Order of operations here matters. 😊 To make this more clear, I would follow the `progress = progress || 5` pattern.